lightkurve/lightcurvefile.py

```diff
@@ -118,7 +114,116 @@ def plot(self, flux_types=None, style='lightkurve', **kwargs):
             for idx, ft in enumerate(flux_types):
                 lc = self.get_lightcurve(ft)
                 kwargs['color'] = np.asarray(mpl.rcParams['axes.prop_cycle'])[idx]['color']
-                lc.plot(label=ft, **kwargs)
+                if method == 'plot':
+                    lc.plot(label=ft, **kwargs)
+                elif method == 'scatter':
+                    lc.scatter(label=ft, **kwargs)
+                elif method == 'errorbar':
+                    lc.errorbar(label=ft, **kwargs)
+
+
+    def plot(self, flux_types=None, style='lightkurve', **kwargs):
```

Comment:
Caution: it looks like the `flux_types` and `style` arguments are not passed to `self._create_plot()` in this function below.  In the `LightCurve` class we accepted these as part of `**kwargs`, we'll want to do the same here.